melodic/distribution.yaml

```diff
@@ -9726,6 +9726,12 @@ repositories:
       url: https://github.com/facontidavide/ros_type_introspection.git
       version: master
     status: developed
+  rosaic:
+    doc:
+      type: git
+      url: https://github.com/septentrio-gnss/rosaic
```

Comment:
This is failing our CI because the URL needs to end with `.git`.

Reply:
Many thanks for pointing this out. I will append the .git momentarily.